infoschema/builder.go

```diff
@@ -211,6 +211,12 @@ func (b *Builder) ApplyDiff(m *meta.Meta, diff *model.SchemaDiff) ([]int64, erro
 		return b.applyDropPolicy(diff.SchemaID), nil
 	case model.ActionAlterPlacementPolicy:
 		return b.applyAlterPolicy(m, diff)
+	case model.ActionCreateResourceGroup:
```

Comment:
There is only applyDiff operation, no full schema load operation?

Reply:
I added it in [5bacb0](https://github.com/pingcap/tidb/pull/40188/commits/5bacb05a56ae703a0bb10bf092bce407a978ac54)